packages/core/src/components/fields/SchemaField.js

```diff
@@ -362,7 +362,9 @@ function SchemaFieldRender(props) {
             onBlur={props.onBlur}
             onChange={props.onChange}
             onFocus={props.onFocus}
-            options={schema.anyOf}
+            options={schema.anyOf.map(_schema =>
```

Comment:
I'm going to label this a breaking change because it technically is (so people can't rely on `$ref`s being present in the `options` parameter in custom `OneOfField` / `AnyOfField`s). We can still merge this in now, though, since 3.x will be the next release.

Can you add a section to the "3.x upgrade guide" file in the docs that describes what people should be aware about regarding the breaking change?

Reply:
Added - please verify if it makes sense to you

Reply:
This looks great, thanks!